VMBackup/main/parameterparser.py

```diff
@@ -102,3 +106,9 @@ def __init__(self, protected_settings, public_settings, backup_logger):
         except Exception as e:
             errorMsg = "Exception occurred while populating includeLunList, Exception: %s" % (str(e))
             backup_logger.log(errorMsg, True)
+        
+        if(self.settings != None):
+            for flag in self.settings.keys():
+                if(flag not in self.wellKnownSettingFlags):
+                    backup_logger.log("The received " + str(flag) + " is not an expected setting name.", True)
```

Comment:
These logs will flow to jarvis, right?

Reply:
The logging mechanism to Kusto is not yet implemented in Linux, when implemented will do the necessary changes.